kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/client/messages/Messages.properties

```
@@ -321,6 +319,20 @@ removeAllAnchorText=Remove All PIDs
 removeAllVisibleAnchorText=Remove All Visible PIDs
 snapshotDownloadSearchBoxPlaceholder=Filter PIDs by name...
 
+deviceSnapshotDownloadModalHint=Select the PIDs for the service configurations that will be downloaded locally.
+deviceWiregraphDownloadModalHint=A partial snapshot containing the current wiregraph configuration will be generated and downloaded.
+formatDownloadHint=The configurations can be exported either as JSON or XML format.
+
+deviceSnapshotDownloadModalDescription=Select the PIDs for the service configurations that will be downloaded locally.
+deviceSnapshotDownloadFormatHint=The configurations can be exported either as JSON or XML format.
+
+deviceSnapshotRollbackConfirm=Select the PIDs for which you wish to execute the configuration rollback. During this operation, the device may experience temporary disconnection and reconnection, depending on the updated services.
+deviceSnapshotRollbackHint=With the Advanced Mode, it is possible to clear the device persistency when applying the rollback. Click on the checkbox for further details.
```

```suggestion
deviceSnapshotRollbackHint=Select Advanced Mode to clear the device persistency when performing the rollback
```

or, as another alternative:

```suggestion
deviceSnapshotRollbackHint=Select Advanced Mode to perform a factory restore of the framework and apply the selected configuration
```